ipyleaflet/leaflet.py

```diff
@@ -782,6 +782,79 @@ class VideoOverlay(RasterLayer):
     attribution = Unicode().tag(sync=True, o=True)
 
 
+class ImageService(Layer):
+    """ImageService class
+
+    Image Service layer for raster data served through a web service
+
+    Attributes
+    ----------
+    url: string, default ""
+        Url to the image service
+    f: string, default "image"
+        response format (use `'image'` to stream as bytes)
+    format: string, default "jpgpng"
+        format of exported image
+    pixel_type: string, default "UNKNOWN"
+        data type of the raster image
+    no_data: list, default []
+        pixel value or comma-delimited list of pixel values representing no data
+    no_data_interpretation: string, default ""
+        how to interpret no data values
```

Comment:
What are possible values?

Reply:
added an enumerated list of possible values